packages/react-native-win32/src/Libraries/Components/ScrollView/ScrollViewWin32.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import RN = require('react-native');
```

Comment:
Going forward, we're hoping to start writing these type of internal components in Flow instead of Typescript, since we won't be able to get accurate React Native typings in TS going forward.

This is all a bit hard since ViewWin32 is already in Typescript, but it might make more sense to start transitioning earlier rather than later.